packages/x-date-pickers-pro/package.json

```
@@ -47,7 +47,7 @@
     "@date-io/luxon": "^2.15.0",
     "@date-io/moment": "^2.15.0",
     "@mui/utils": "^5.9.3",
-    "@mui/x-date-pickers": "5.0.0-beta.7",
+    "@mui/x-date-pickers": "5.0.0",
     "@mui/x-license-pro": "5.12.1",
```

Maybe we should also keep bumping the `licence` version here as well as in data-grid packages? 🤔 
```suggestion
    "@mui/x-license-pro": "5.17.0",
```

---

Because now, if I understand correctly, given a user with `pro pickers` package and `pro` or `premium` data grid package - we would force two versions of `x-license-pro` installations 😢